core/src/main/java/com/linecorp/armeria/server/AbstractVirtualHostBuilder.java

```diff
@@ -283,75 +281,75 @@ B service(T serviceWithPathMappings, Function<T, R> decorator) {
      * Binds the specified annotated service object under the path prefix {@code "/"}.
      */
     public B annotatedService(Object service) {
-        return annotatedService("/", service);
+        return annotatedService("/", service, ImmutableList.of(), Function.identity());
     }
 
     /**
      * Binds the specified annotated service object under the path prefix {@code "/"}.
      */
-    public B annotatedService(
-            Object service,
-            Function<Service<HttpRequest, HttpResponse>,
-                     ? extends Service<HttpRequest, HttpResponse>> decorator) {
-        return annotatedService("/", service, decorator);
+    public B annotatedService(Object service,
+                              Iterable<?> exceptionHandlersAndConverters) {
```

Comment:
I wonder what happens if we add a version that accepts `Object...` instead of `Iterable<?>`. Compilation errors?

Reply:
I will try to do it.